faq/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page_full
+title: FAQ
+---
+
+**Q: When is the next meetup?**

```

Comment:
Can you mark up the questions as h1 or h2 (`# ` or `## `)?

Reply:
Sure, done